hub-chart/templates/postcreatehooks/wds.yaml

```diff
@@ -0,0 +1,223 @@
+      verbs:
+      - get
+      - list
+      - watch
+      - create
+      - patch
+      - update
+    - apiGroups:
+      - tenancy.kflex.kubestellar.org
+      resources:
+      - controlplanes
+      verbs:
+      - get
+      - list
+      - watch
+    - apiGroups:
+      - tenancy.kflex.kubestellar.org
+      resources:
+      - controlplanes/status
+      verbs:
+      - get
+      - patch
+      - update
+    - apiGroups:
+      - authentication.k8s.io
+      resources:
+      - tokenreviews
+      verbs:
+      - create
+    - apiGroups:
+      - authorization.k8s.io
+      resources:
+      - subjectaccessreviews
+      verbs:
+      - create
+    - apiGroups:
+      - rbac.authorization.k8s.io
+      resources:
+      - clusterroles
+      - clusterrolebindings
+      verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+    - apiGroups:
+      - rbac.authorization.k8s.io
+      resources:
+      - roles
+      - rolebindings
+      verbs:
+      - get
+      - list
+      - watch
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: "{{"{{.Namespace}}-{{.HookName}}"}}"
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: "{{"{{.Namespace}}-{{.HookName}}"}}"
+    subjects:
+    - kind: ServiceAccount
+      name: default
+      namespace: "{{"{{.Namespace}}"}}"
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: Role
+    metadata:
+      name: "{{"{{.HookName}}"}}"
+    rules:
+    - apiGroups:
+      - ""
+      resources:
+      - configmaps
+      verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+    - apiGroups:
+      - coordination.k8s.io
+      resources:
+      - leases
+      verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+    - apiGroups:
+      - ""
+      resources:
+      - events
+      - secrets
+      - services
+      verbs:
+      - create
+      - patch
+      - update
+    - apiGroups:
+      - ""
+      resources:
+      - serviceaccounts
+      verbs:
+      - get
+      - create
+    - apiGroups:
+      - ""
+      resources:
+      - services
+      verbs:
+      - get
+    - apiGroups:
+      - apps
+      resources:
+      - deployments
+      verbs:
+      - get
+      - create
+    - apiGroups:
+      - rbac.authorization.k8s.io
+      resources:
+      - roles
+      - rolebindings
+      verbs:
+      - create
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: RoleBinding
+    metadata:
+      name: "{{"{{.HookName}}"}}"
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: Role
+      name: "{{"{{.HookName}}"}}"
+    subjects:
+    - kind: ServiceAccount
+      name: default
+      namespace: "{{"{{.Namespace}}"}}"
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: "{{"{{.HookName}}"}}"

```

Comment:
Perhaps add "kscm" into this name, to indicate specifically that it is for the KubeStellar controller-manager. I suggest something short because of the limit on Job name length (https://kubernetes.io/docs/concepts/workloads/controllers/job/#writing-a-job-spec).

Reply:
The hook name will be already `its` ... I think it's appropriate not to be specific of what is in it now and in the future, in my opinion

Reply:
This is WDS, not ITS.

This Job is not part of the user interface. I suggest naming this job in a way that helps someone semi-knowledgeable trying to debug something. Remember, for example, that a Job leaves a Pod lying around, some it is not at all uncommon for someone to get a listing of all the Pods. The purpose of this Job is to install the Helm chart for the kubestellar-controller-manager. I think that a name like "install-kubestellar-controller-manager" or "instantiate-kubestellar-controller-manager-chart" would be understandable.

This is not a blocker for this PR.